### PR TITLE
Add styles for hero component

### LINKED ISSUE #61: Add styles for hero component
- Storybook using HTML structure from dimes. See styles in dimes `_hero.scss`
- Note class names changed for BEM conformance

### PR DESCRIPTION
**Pull request by bonniegee**
Fixes #61 

- Adds the hero scss from DIMES
- No changes made when copying from DIMES, though the line spacing (in the hero text) does not look correct to me--I'm not sure if this is an issue?
- I added very minimal comments and would welcome feedback on what else would be useful to include in comments

### CODE REVIEW

**Comment by helrond** [comment]
> No changes made when copying from DIMES, though the line spacing (in the hero text) does not look correct to me--I'm not sure if this is an issue?

I think there might be some styles for the typography in `_typography.scss` in DIMES. Initially we were putting all typography in one place, but that ultimately became more confusing than helpful as we got more components in place.

**Comment by helrond** [comment]
Looks good to me! I'm wondering if we should do away with the top margin on the `.hero` class? That seems variable depending on the context the component is used in, right?

File: stylesheets/components/_hero.scss [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  align-items: center;
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  padding: 30px;
+
+  @include md-up {
+    flex-wrap: nowrap;
+    justify-content: center;
+    margin-top: 113px;
```
Comment (helrond) [inline stylesheets/components/_hero.scss]: Let's pull the margin-top out of the responsive styles as well.

File: stylesheets/components/_hero.scss [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  }
+
+  @include lg-up {
+    margin-top: 103px;
```
Comment (helrond) [inline stylesheets/components/_hero.scss]: Here too.

**Review by helrond — APPROVED** [review]
A few more `margin-top` properties to remove...

**Review by HaSistrunk — APPROVED** [review]
Looks good!

**Review by p-galligan — APPROVED** [review]

**Pull request merged**